scripts/spectrogram_advoc.py

```diff
@@ -6,9 +6,19 @@
   import numpy as np
   import os
   from tqdm import tqdm
-
+  import tensorflow as tf
   from advoc.audioio import save_as_wav
-  from advoc.spectral import r9y9_melspec_to_waveform
+  from advoc.spectral import r9y9_melspec_to_waveform, magspec_to_waveform_lws
+  from advoc.spectral import create_inverse_mel_filterbank
+
+  def tacotron_mel_to_mag(X_mel_dbnorm, invmeltrans):
```

Comment:
This is a little bit ugly here. Can you add a TODO to move this code into the main `advoc` library?

Reply:
added TODO.